query_test.go

```diff
@@ -594,6 +597,8 @@ func TestQueryEffectiveBufferSizeHeterogeneous(t *testing.T) {
 	assert.Nil(t, err)
 	_, err = query.SetBuffer("cols", colsWrite)
 	assert.Nil(t, err)
+	_, _, _, err = query.SetBufferVarNullable("a2", a2OffWrite, a2DataWrite, a2Validity)
```

Comment:
Add a set for `SetBufferNullable`/`BufferSizeVarNullable` too?

Reply:
Good idea, done